src/main/java/ch/wisv/areafiftylan/seats/model/Seat.java

```diff
@@ -39,7 +39,7 @@
     private Long Id;
 
     @JsonView(View.Public.class)
-    private boolean taken = false;
```

Comment:
We do use that property in the front end, I have created an issue there detailing the needed changes.